app/api/v2/handlers/operation_api.py

```diff
@@ -34,39 +34,53 @@ def add_routes(self, app: web.Application):
+                                      'UUID).  Use fields from the `BaseGetOneQuerySchema` in the request body to add '
+                                      '`include` and `exclude` filters.')
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
     @aiohttp_apispec.response_schema(OperationSchema(partial=True))
     async def get_operation_by_id(self, request: web.Request):
         operation = await self.get_object(request)
         return web.json_response(operation)
 
     @aiohttp_apispec.docs(tags=['operations'],
-                          summary='Required nested schema fields are as follows: "adversary.adversary_id", '
-                                  '"planner.planner_id", and "source.id".')
-    @aiohttp_apispec.request_schema(OperationSchema())
+                          summary='Create a new CALDERA operation record',
+                          description='Create a new CALDERA operation using the format provided in the '
+                                      '`OperationSchema`. Required nested schema fields are as follows: '
+                                      '"adversary.adversary_id", "planner.planner_id", and "source.id"')
+    @aiohttp_apispec.request_schema(OperationSchema)
     @aiohttp_apispec.response_schema(OperationSchema)
     async def create_operation(self, request: web.Request):
         operation = await self.create_object(request)
         return web.json_response(operation.display)
 
-    @aiohttp_apispec.docs(tags=['operations'])
-    @aiohttp_apispec.request_schema(OperationSchema(partial=True, only=['state',
-                                                                        'autonomous',
-                                                                        'obfuscator']))
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Update fields within an operation',
+                          description='Update one CALDERA operation in memory based on the operation id (String '
+                                      'UUID). Any fields in the operation object may be edited in the request body '
+                                      'using the `OperationSchema`.')
+    @aiohttp_apispec.request_schema(OperationSchema(partial=True))
     @aiohttp_apispec.response_schema(OperationSchema(partial=True))
     async def update_operation(self, request: web.Request):
         operation = await self.update_object(request)
         return web.json_response(operation.display)
 
-    @aiohttp_apispec.docs(tags=['operations'])
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Delete an operation by operation id',
+                          description='Delete one CALDERA operation from memory based on the operation id (String '
+                                      'UUID).')
     @aiohttp_apispec.response_schema(OperationSchema)
```

Comment:
One last thing that might be nice to have is a description for the `response_schema` in every endpoint. While GET operation by ID, PUT, PATCH, and POST all return a single dumped Operation object, `GET /operations` and `DELETE /operations{id}` return a list of dumped objects and a 204 HTTP response code, respectively.